Chrysalis-Papilio/Soen490-Papilio-Website
# Refactor

## General Information

**Issue number**: Papilio-66

**Task description**:
Separate the name input into 2, firstName and lastName
- Add new inputs
- Replace old test with new one with the new inputs

Improve code coverage to respect the set threshold.

## Manual Testing
To test the modification you have to create your own business to access the employee form.

Make sure that you have the appropriate variables in the backend .env file. (Contact me if you don't)
1. start the backend
```npm run build && npm start```
2. start the website
```npm start```
3. on the main page add the businessId then click on create business
4. enter the profile information then click next
5. enter the admin root account information then click next
6. click submit

The form should be submitted successfully

## Testing

To run all the unit tests, run the following code: ```npm test```
To make sure that the tests have reach the appropriate level of coverage, run the following code: ```npm run coverage```
